.travis.yml

```diff
@@ -38,7 +38,7 @@ python:
 - 3.4
 - pypy
 install:
-- pip install tox-travis
+- pip install tox-travis --use-mirrors
```

Comment:
This option is deprecated, and removed from the version of pup that is installed in the '3.5' image.